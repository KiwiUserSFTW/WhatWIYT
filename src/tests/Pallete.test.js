import React from "react";
import renderer from "react-test-renderer";
import Pallete from "../components/UI/Pallets";
import TestProvider from './tests_tools/TestProvider'
import { initialFakeData } from "./tests_tools/initialFakeStates";
test("DialogInfo was correctly", () => {
    const tree = renderer
        .create(
            <TestProvider>
                <Pallete data={initialFakeData} />
            </TestProvider>
        )
        .toJSON();

    expect(tree).toMatchSnapshot();
});