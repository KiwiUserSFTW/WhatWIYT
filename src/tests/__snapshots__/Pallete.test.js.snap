// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialogInfo was correctly 1`] = `
Array [
  <div
    style={
      Object {
        "borderRadius": "10px",
        "margin": "10px",
        "minWidth": "15%",
      }
    }
  >
    <div
      className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-bhp9pd-MuiPaper-root-MuiCard-root"
    >
      <div
        className="MuiCardContent-root css-46bh2p-MuiCardContent-root"
      >
        <h6
          className="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom css-137rogq-MuiTypography-root"
        >
          Berlin
        </h6>
        <p
          className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          18Â°C
           
          <br />
          clear sky
        </p>
      </div>
      <div
        className="MuiCardActions-root MuiCardActions-spacing css-1t6e9jv-MuiCardActions-root"
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall css-la9hut-MuiButtonBase-root-MuiButton-root"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          More info
        </button>
      </div>
    </div>
  </div>,
  " ",
]
`;
